fcrepo-api-x-registry/src/main/java/org/fcrepo/apix/registry/ConfigHelper.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.apix.registry;
+
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
+
+/**
+ * Helps get properties from OSGi ConfigurationAdmin.
+ *
+ * @author apb@jhu.edu
+ */
+public class ConfigHelper {
+
+    private ConfigurationAdmin configAdmin;
```

Comment:
Well, there's a [maven plugin](http://aries.apache.org/modules/blueprint-maven-plugin.html) that will _generate_ a blueprint file, and it looks like there is a [service](http://aries.apache.org/modules/blueprintannotation.html) being prototyped that'll directly do injection based on annotations.  It's unclear to me if/how either of those work in practice.  To be honest, I'd love to just smile slightly, and back away slowly from blueprint without any sudden moves.